scripts/versions.sh

```diff
@@ -2,9 +2,28 @@
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
-# Set up the versions to be used
-AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.3'}
-SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.8'}
+set -e
+set -o pipefail
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+
+function setGO_VERSION() {
```

Comment:
Nit: Should the capitalization here be `setGoVersion` to be consistent with `getDepVersion`?

Reply:
The idea is that `GO_VERSION` is a specific env var, whereas `getDepVersion` takes a param.

Reply:
Gotcha, it seems a little odd given that we haven't used that convention anywhere else, but I don't feel strongly either way.